### PR TITLE
test: Add docker exec integration test.

### LINKED ISSUE #169: Add docker exec test
Add docker functionality exec test

### PR DESCRIPTION
**Pull request by GabyCT**
Add docker integration for exec command.

Fixes #169

Signed-off-by: Gabriela Cervantes <gabriela.cervantes.tellez@intel.com>

### CODE REVIEW

**Comment by devimc** [comment]
lgtm

File: integration/docker/exec_test.go [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+		It("should not exist", func() {
+			args = []string{id, "sh", "-c", "ls /etc/foo >/dev/null"}
+			stdout, stderr, exitCode = dockerExec(args...)
+			Expect(exitCode).To(Equal(1))
+			Expect(stdout).To(BeEmpty())
+			Expect(stderr).ToNot(BeEmpty())
+		})
+	})
+
+	DescribeTable("check exec honours '--user'",
```
Comment (grahamwhaley) [inline integration/docker/exec_test.go]: nice thing to check :-)

**Comment by GabyCT** [comment]
@jcvenegas could you please check this?

**Comment by chavafg** [comment]
lgtm

**Pull request merged**